applications/tari_console_wallet/src/automation/command_parser.rs

```diff
@@ -128,8 +128,8 @@ pub fn parse_command(command: &str) -> Result<ParsedCommand, ParseError> {
         CoinSplit => parse_coin_split(args)?,
         DiscoverPeer => parse_public_key(args)?,
         Whois => parse_whois(args)?,
-        ExportUtxos => parse_export_utxos(args)?, // todo: only show X number of utxos
-        ExportSpentUtxos => parse_export_spent_utxos(args)?, // todo: only show X number of utxos
```

Comment:
I added these todo's and I don't really think they are useful for these commands

Reply:
Yep just a nice to have but not important